packages/lesswrong/themes/lesswrongTheme.js

```diff
@@ -37,7 +37,7 @@ const palette = {
     main: '#5f9b65',
   },
   lwTertiary: {
-    main: "#607e88"
+    main: "#3b839a"
```

Comment:
This is significantly changing the color of a lot of things, including comment permalinks and Load More links. Is that intended?